mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -815,4 +829,14 @@ public boolean hasChildren(MailboxPath mailboxPath, MailboxSession session) thro
         Mailbox mailbox = mapper.findMailboxByPath(mailboxPath);
         return mapper.hasChildren(mailbox, session.getPathDelimiter());
     }
+
+    @Override
+    public void setLimitOfAnnotations(int limitOfAnnotations) {
```

Comment:
But there are so many parameters in constructor. Is there any convention for that? Eg: the number of function's parameters must less than 10
And one more thing, the StoreMailboxManager is main class so if we changed the constructor (with adding 2 new parameter just for annotation) will effect a lot, so many code should be changed. That seems not good I think. WDYT